src/common/components/Header/MenuItem/style.css.ts

```diff
@@ -0,0 +1,20 @@
+import { style } from '@vanilla-extract/css';
+
+import { theme } from 'styles/theme.css';
+
+export const menuItem = style({
+  ...theme.font.HEADING_6_18_B,
+  color: theme.color.baseText,
+});
+
+export const menuLink = style({
+  selectors: {
+    '&:hover:not([disabled])': {
+      textDecoration: `underline ${theme.color.primary} 2px`,
+      textUnderlineOffset: 21,
+    },
+    '&.active': {
+      color: theme.color.primary,
+    },
+  },
+});
```

Comment:
```suggestion
export const menuLink = style({
  textDecoration: 'underline',
  textDecorationThickness: 2,
  textDecorationColor: 'transparent',
  textUnderlineOffset: 21,
  transition: 'all 0.3s ease',

  /*add those for opera and mozilla support*/
  WebkitTextDecorationColor: 'transparent',
  MozTextDecorationColor: 'transparent',
  selectors: {
    '&:hover:not([disabled])': {
      textDecorationColor: theme.color.primary,
      WebkitTextDecorationColor: theme.color.primary,
      MozTextDecorationColor: theme.color.primary,
    },
    '&.active': {
      color: theme.color.primary,
    },
  },
});
```

위와 같이 수정하면 transition 효과를 줄 수 있어요~ [참고자료](https://stackoverflow.com/questions/30352431/css-transition-not-working-with-underline/53139326#53139326)
기본적으로 모든 애니메이션에는 transition을 주면 좋을 거 같습니다~~

Reply:
애니메이션 좋습니다! 
혹시 크로스 브라우징은 작업 전반에 있어 고려하실 예정인가요?  (webkit, moz 매번 별도로 추가해주는 것)

Reply:
추가해주면 좋을 거 같긴 한데 찾는데 리소스 많이 든다 싶으면 빼도 상관 없습니다~
현재 같은 경우는 underline에 transition 적용 방법 찾아보니 mozilla, opera 에 대한 처리도 같이 알려줘서 추가해 봤습니다!

Reply:
아하 좋습니다! 
제 의견은, 크로스 브라우징을 고려하는 것은 너무 좋으나 **현재 상황상** 매번 이를 고려하며 명시해주는 것은 리소스 대비 효율이 너무 안난다고 생각되어요. 찾아보니 **브라우저 별 css prefix를 자동으로 추가해주는 autoprefix 툴**들이 많은 것 같은데, 우선 브라우저 prefix는 간과하고 구현 후, 기능 완성 이후에 크로스브라우징 일괄 처리하는건 어떻게 생각하시나요??

Reply:
와웅 너무 좋은 거 같습니다~~~~~

Reply:
애니메이션 반영했습니다! 근데 뭔가 아주 사알짝 답답(?)한 느낌이 있는 것 같은데, 그래도 0.3s로 통일하기로 했다면 통일하는게 좋을까요? ㅠㅠ 

https://github.com/sopt-makers/sopt-recruiting-frontend/assets/81505421/cfdde5fb-4d85-481c-b2bb-528421e2e2ea

Reply:
아뇨!! 답답하면 그에 맞춰 수정하는 게 좋을 거 같아요~~~
0.3s로 고정된 건 아니라서요!!!

Reply:
0.2s로 줄이고 
답답한 느낌이 시작(fade-in)속도 때문인 것 같아서 `ease -> ease-out`으로 바꿔서 fade-in 속도 높여주었습니다! 

https://github.com/sopt-makers/sopt-recruiting-frontend/assets/81505421/86b19b20-5a56-4455-8c45-52c104aadf66

괜찮으시면 따봉 확인 후 리졸브 하겠슴다